includes/notes/class-wc-payments-notes-set-up-stripelink.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Set up Stripe Link note for WooCommerce inbox.
+ *
+ * @package WooCommerce\Payments\Admin
+ */
+
+use Automattic\WooCommerce\Admin\Notes\NoteTraits;
+use WCPay\Payment_Methods\Link_Payment_Method;
+use WCPay\Payment_Methods\CC_Payment_Method;
+
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class WC_Payments_Notes_Setup_StripeLink
+ */
+class WC_Payments_Notes_Set_Up_StripeLink {
+	use NoteTraits;
+
+	/**
+	 * Name of the note for use in the database.
+	 */
+	const NOTE_NAME = 'wc-payments-notes-set-up-stripe-link';
+
+	/**
+	 * CTA button link
+	 */
+	const NOTE_DOCUMENTATION_URL = 'https://woocommerce.com/document/payments/woocommerce-payments-stripe-link/';
+
+	/**
+	 * The account service instance.
+	 *
+	 * @var WC_Payment_Gateway_WCPay
+	 */
+	private static $gateway;
+
+	/**
+	 * Checks if a note can and should be added.
+	 *
+	 * @return bool
+	 */
+	public static function should_display_note() {
```

Comment:
Can we add return type here?
```suggestion
	public static function should_display_note():bool {
```

Reply:
Done.